### ISSUE TITLE
Need FileNode.getNode method

### ISSUE THREAD

**Issue by J-F-Liu** [twistedfall/opencv-rust#78]
```
operator[]()
FileNode cv::FileNode::operator[]  ( const char * nodename ) const

Python:
retval	=	cv.FileNode.at(	i	)
retval	=	cv.FileNode.getNode(	nodename	)
```

**Comment by twistedfall** [comment]
I see, for now those will need to be implemented manually as we don't support automatic handling of `operator[]`, I'll see what can be done

**Comment by twistedfall** [comment]
Fixed in v0.30.0. Please note that this version comes with the totally rewritten binding generator and you will need llvm installed instead of python to be able to correctly regenerate bindings. Please see the [changelog](https://github.com/twistedfall/opencv-rust/blob/master/CHANGES.md) and [readme](https://github.com/twistedfall/opencv-rust/blob/master/README.md) for more details.
